drivers/soundwire/amd_manager.c

```diff
@@ -926,6 +926,7 @@ static int amd_sdw_manager_probe(struct platform_device *pdev)
 	amd_manager->bus.compute_params = &amd_sdw_compute_params;
 	amd_manager->bus.clk_stop_timeout = 200;
 	amd_manager->bus.link_id = amd_manager->instance;
+	amd_manager->bus.controller_id = amd_manager->instance;
```

Comment:
Currently we are populating one controller and two links under same controller scope.
for that we expect amd_manager->bus.controller_id should be set to zero. We can't populate as two separate controllers due to same design opted in other OS which can't be modified due to some design constraints.

In future any platform requires new design , then we will pass the controller id as driver data.

Reply:
Please modify the commit message as well.

Reply:
thanks for the quick feedback @vijendarmukunda, much appreciated.